nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-api/src/main/java/org/nuxeo/ecm/platform/audit/api/AuditReader.java

```diff
@@ -40,6 +40,16 @@
      */
     List<LogEntry> getLogEntriesFor(String uuid);
```

Comment:
Please mark in the interface the old method with the `@Deprecated` annotation, and a `@deprecated since 8.4, use xxx instead` in the Javadoc.